features/schedule/basic_workflow/feature.go

```diff
@@ -0,0 +1,122 @@
+package basic_workflow
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows: BasicScheduleWorkflow,
+	Execute:   Execute,
+}
+
+func BasicScheduleWorkflow(ctx workflow.Context, arg string) (string, error) {
+	return arg, nil
+}
+
+func Execute(ctx context.Context, r *harness.Runner) (client.WorkflowRun, error) {
+	// Start schedule every 2s
+	workflowID := uuid.NewString()
+	handle, err := r.Client.ScheduleClient().Create(ctx, client.ScheduleOptions{
+		// TODO(cretz): Required? https://github.com/temporalio/sdk-go/issues/957
+		ID:      uuid.NewString(),
+		Spec:    client.ScheduleSpec{Intervals: []client.ScheduleIntervalSpec{{Every: 2 * time.Second}}},
+		Overlap: enums.SCHEDULE_OVERLAP_POLICY_BUFFER_ONE,
+		// TODO(cretz): Would prefer if this were a pointer. See
```

Comment:
Issue has been fixed, this need to be a pointer.

Reply:
It is not resolved in a released version though. Cannot update.